Source/diablo.cpp

```diff
@@ -1720,6 +1722,8 @@ void game_logic()
 	CheckQuests();
 	drawpanflag |= 1;
 	pfile_update(FALSE);
+
+	plrctrls_game_logic();

```

Comment:
Maybe it would make more sens to have this next to CheckCursMove? Also it might need to take 60fps in to account: https://github.com/diasurgical/devilutionX/commit/2b06fb1a6d83f290db0fddd280a1fa5a5dfba0fa

Reply:
Maybe, I'm not entirely sure how often this is supposed to run.

Reply:
If it affects item highlighting or pointer movment then it should run before every visual frame, if it updates game logic (it shoudn't) it should only update at 20hz

Reply:
Done, at the end of `CheckCursMove` turned out to be the perfect spot.